guillotina/fields/patch.py

```diff
@@ -43,6 +43,8 @@ def field_converter(field, value, context):
             raise ValueDeserializationError(
                 field, value, f'"{operation_name}" not a valid operation')
         value = operation(context, value.get('value'))
+    else:
```

Comment:
Can we change this to maybe `elif isinstance(value, (dict, list)):`?

Reply:
sure!

Reply:
Why int?

Reply:
@vangheem ups, forget this :P